tests/unit/test_svs_tiered.cpp

```diff
@@ -169,7 +169,8 @@ TYPED_TEST(SVSTieredIndexTest, ThreadsReservation) {
     }
 
     std::chrono::milliseconds timeout{1000}; // long enough to reserve all threads
-    SVSParams params = {.type = TypeParam::get_index_type(), .dim = 4, .metric = VecSimMetric_L2};
+    SVSParams params = {
```

Comment:
why?

Reply:
so `num_threads` two lines above is only used to verify the system has enough threads to run the test?
if so can we change to 
``` 
const auto num_threads =  getAvailableCPUs();
```
?

Reply:
sorry i confused `params.num_threads` and `mock_thread_pool.thread_pool_size`